src/main/pages/setup-openshift/openshift-admin-guide.adoc

```diff
@@ -362,37 +348,41 @@ oc delete pvc -l=app=postgres
 ----
 
 [id="create-workspace-objects-in-personal-namespaces"]
-== Create workspace objects in personal namespaces
+== Creating workspace objects in personal namespaces
 
 You can register the OpenShift server as an identity provider when Che is installed in multi-user mode. This allows you to create workspace objects in the OpenShift namespace of the user that is logged in Che through Keycloak.
 
 To create a workspace object in the namespace of the user that is logged into Che:
 
-* link:#openshift-identity-provider-registration[Register], inside Keycloak, an OpenShift identity provider that points to the OpenShift console of the cluster
-* link:#che-configuration[configure] Che to use this Keycloak identity provider in order to retrieve the OpenShift tokens of Che users.
+* link:#openshift-identity-provider-registration[Register], inside Keycloak, an OpenShift identity provider that points to the OpenShift console of the cluster.
+* link:#che-configuration[Configure] Che to use the Keycloak identity provider to retrieve the OpenShift tokens of the Che users.
 
-Once this is done, every interactive action done by a Che user on workspaces, such as start or stop, will create OpenShift resources under his personal OpenShift account. And the first time the user will try to do it, he will be asked to link his Keycloak account with his personal OpenShift account: which he can do by simply following the provided link in the notification message.
+Every workspace action such as start or stop creates an OpenShift resource in the OpenShift user account.  A notification message displays which allows you to link the Keycloak account to your OpenShift user account.
 
-But for non-interactive workspace actions, such as workspace stop on idling or Che server shutdown, the account used for operations on OpenShift resources will fall back to the dedicated OpenShift account configured for the Kubernetes infrastructure, as described in the link:admin-guide#who-creates-workspace-objects[AdminGuide].
+But for non-interactive workspace actions, such as workspace stop on idling or Che server shutdown, the dedicated OpenShift account configured for the Kubernetes infrastructure is used. See link:admin-guide#who-creates-workspace-objects[AdminGuide] for more information.
```

Comment:
* incorrect URL; should be `openshift-admin-guide.html#who-creates-workspace-objects`
* "AdminGuide" -> "Setting up the project workspace"